config/initializers/devise.rb

```diff
@@ -5,9 +5,8 @@
   config.case_insensitive_keys = [:email]
   config.strip_whitespace_keys = [:email]
   config.skip_session_storage = [:http_auth]
-  config.stretches = Rails.env.test? ? 1 : 10
+  config.stretches = Rails.env.test? ? 1 : 12
   config.pepper = '0761ef0892f647b6d3c525ee7d98043efadaae4b42d77102b56beba91bd44360d4cfb0ffa713778c281afe99e18c22ed0364330c28c0292fc25251c3f8858331'
-  config.secret_key = '7dce324450fa7e8371325c63e65afb6c14fcde6f14ab5577d16d91d0a0f590e643219d31fbbe0e314d1888a0baa3aa3393a3d7870b4ff02a9e32cbcb2febe0fa'
```

Comment:
Yikes. This is AFAIK a real secret.

If you don't set it, Devise uses `secret_key_base`. Changing it now means we invalidate users' outstanding password-reset/unlock tokens. But, given we don't have a working mail provider, no one has any of those they can use.